torchprime/torch_xla_models/configs/default.yaml

```diff
@@ -25,8 +25,25 @@ optimizer:
 lr_scheduler:
   type: linear
   warmup_steps: 0
+
+# The virtual device mesh shape to use within a TPU slice. This is also called
+# the "ICI mesh", since devices within a slice enjoy a faster network called
+# "Inter-Chip Interconnect".
 mesh:
```

Comment:
nit: Should we just rename this to ici_mesh?

Reply:
Renamed